cmd/api/errors.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"bookshop/internal/utils"
+	"log/slog"
+	"net/http"
+)
+
+func (app *application) errorResponse(w http.ResponseWriter, r *http.Request, status int, message string) {
+	if err := utils.WriteJSON(w, status, message, nil); err != nil {
```

Comment:
If we receive error we will receive 200 response by default. 

Add `w.WriteHeader(500)` in this if clause.